layer/logged_data/loggers.py

```diff
@@ -0,0 +1,357 @@
+from argparse import Namespace
+from pathlib import Path
+from typing import Any, Dict, Generator, List, Mapping, MutableMapping, Optional, Union
+
+import torch
+
+import layer
+
+
+class LayerLogger("pytorch_lightning.loggers.logger.Logger"):
```

Comment:
Can we call this class `PytorchLightningLayerLogger`?

Reply:
Done